supervisor/process_child.go

```diff
@@ -0,0 +1,60 @@
+// pmm-agent
+// Copyright (C) 2018 Percona LLC
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+// +build ignore
+
+// Run it with:
+//   go run -tags child process_child.go
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/percona/pmm-agent/supervisor"
+	"github.com/percona/pmm/api/agent"
```

Comment:
Move that import to the second group. The last group is only for the packages from this repo.
(I wonder why golangci-lint is not complaining about this…)